contrib/rust/tests/python/pants_test/contrib/rust/utils/test_custom_build_output_parsing.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import unittest
+
+from pants.contrib.rust.utils.custom_build_output_parsing import parse_multiple_cargo_statements
+
+
```

Comment:
Adding (or even replacing one unit test with) one integration test (extending `PantsRunIntegrationTest`) would be very valuable. Doing it as part of adding the example usage you mentioned (in `contrib/rust/examples` maybe?) would be awesome.

(These tests look great, but my general bias is toward ensuring that integration tests are present before diving too deeply into unit tests, since they minimize the dependence on implementation details.)